Tests/SonarScanner.MSBuild.Common.UnitTests/AnalysisConfig/AnalysisConfigTests.cs

```diff
@@ -191,9 +191,19 @@ public void AnalysisConfig_ExpectedXmlFormat()
   <AnalyzersSettings>
     <AnalyzerSettings>
       <RuleSetFilePath>d:\ruleset path.ruleset</RuleSetFilePath>
-      <AnalyzerAssemblyPaths>
-        <Path>c:\analyzer1.dll</Path>
-      </AnalyzerAssemblyPaths>
+      <AnalyzerPlugins>
```

Comment:
This shows what the new format looks like. It has enough information so that the _GetAnalyzerSettings_ task can work out which files to return for test projects that should only have the SonarC#/VB.NET analyzers run against them.